specification/storage/resource-manager/Microsoft.Storage/stable/2023-04-01/storagetasks.json

```diff
@@ -0,0 +1,863 @@
+{

```

Comment:
Why the file is here？
As I Storage Task has changed name to Storage Action, and added to a new RP: https://github.com/Azure/azure-rest-api-specs/tree/main/specification/storageactions/resource-manager